tests/test_v1.py

```diff
@@ -289,17 +289,32 @@ def test_job_position(mongo_app):
     """Ensure initial job state is set to 'waiting'"""
     app, _ = mongo_app
     job_data = {"job_queue": "test"}
-    # Place a job on the queue
+    # Place a 3 jobs on the queue
```

Comment:
Just a very minor fix; I guess we don’t need “a” before “3 jobs” there

Reply:
Thanks! I'm failing at english today :)